vtex-apps/wordpress-integration
# [WORDPRESS-52] Allow video and source tags and style for divs

**What problem is this solving?**
The integration was not allowing the tags video and source, now those are allowed, also the divs was not allowing style attribute. 
[This shoulder store blog post](https://blogwordpress--shoulder.myvtex.com/blog/os-Desejos-do-momento) has this html to render a video:
```
<div style=\"width: 1920px;\" class=\"wp-video\">\n<video class=\"wp-video-shortcode\" id=\"video-2256-1\" width=\"1920\" height=\"1080\" preload=\"metadata\" controls=\"controls\"><source type=\"video/mp4\" src=\"http://blog.shoulder.com.br/wp-content/uploads/2022/02/FF-30s.mp4?_=1\" /><a href=\"http://blog.shoulder.com.br/wp-content/uploads/2022/02/FF-30s.mp4\">http://blog.shoulder.com.br/wp-content/uploads/2022/02/FF-30s.mp4</a></video></div>
```

**How should this be manually tested?**
[Go to this workspace](https://blogwordpress--shoulder.myvtex.com/blog/os-Desejos-do-momento) and search for the video with id `video-2256-1`, you should see the video in the page.
**Screenshots or example usage:**
![Screen Shot 2022-05-26 at 9 47 57](https://user-images.githubusercontent.com/49737670/170512892-fe7c5b3e-4f3e-4fe3-b441-6af2d7dea46f.png)
